tools/build/wheel/split_wheel.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import sys
+import subprocess
+
+import logging
+from pathlib import Path
+from typing import List, Dict, Optional
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+# TODO: Add a function to install dependencies from requirements.txt
+# TODO: Add a function to do submodule setup
+
+
+def get_git_root() -> Path:
```

Comment:
You need project root, rather than a git root (as this command should ideally work if someone downloads our release tarballs), I.e. it should be computed as `PATH(__file__).parent.parent.parent.parent`